golang/scripts/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
I've just use it, because I'm not pretty sure if we should run the Lambda with this script

Reply:
yep, that's correct - you'd need a test or Kurtosis Interactive to actually run the Lambda. which, if the users want, they could stick a testsuite inside their Lambda project... but we don't need to do that